.envrc

```diff
@@ -15,3 +15,6 @@ use_trunk.io() {
 }
 
 use trunk.io
+
+# Needed by `rust-analyzer`. Hash should be the same for all platforms.
+export RUST_SRC_PATH="$HOME/.cache/trunk/tools/rust/2024-05-01-d4daa7f4db5f9d74e00218df573de113/rust-src-nightly/rust-src/lib/rustlib/src/rust/library"
```

Comment:
There doesn't seem to be a way to have Rust components added via `trunk` other than referring to them like this. Luckily, this isn't platform dependent and the path can be found by `cat "$(which rust-src)"` since `trunk` creates a wrapper script to the actual location.